solara/website/components/contact.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+import requests
+
+from typing import Any, Dict, Optional
+import solara
+
+
+postmark_api_key = None
+contact_email_address = None
+
+try:
+    postmark_api_key = os.environ["POSTMARK_API_KEY"]
+    contact_email_address = os.environ["SOLARA_CONTACT_EMAIL_ADDRESS"]
+except Exception:
+    pass
+
+
+@solara.component
+def Contact(
+    style: Dict[str, Any] = {},
+    title="Contact Us",
+    subtitle="We'd love to hear from you!",
+    submit_label="Submit",
+    email_subject="Contact Form Submission",
+):
+    first_name = solara.use_reactive("")
+    last_name = solara.use_reactive("")
+    email = solara.use_reactive("")
+    company = solara.use_reactive("")
+    message = solara.use_reactive("")
+    error: solara.Reactive[Optional[str]] = solara.use_reactive(None)
+
+    def send(*_ignore):
+        if postmark_api_key is None or contact_email_address is None:
+            error.set("Email service not properly configured. Please contact the site administrator at solara@widgetti.io.")
+        else:
+            # Create the email content
+            msg = {}
+            msg["From"] = contact_email_address
+            msg["To"] = contact_email_address
+            msg["Subject"] = email_subject
+            msg["ReplyTo"] = email.value
+
+            # Email body
+            msg["HtmlBody"] = f"""
+            <b>First Name</b>: {first_name.value}<br />
+            <b>Last Name</b>: {last_name.value}<br />
+            <b>Email</b>: {email.value}<br />
+            <b>Company</b>: {company.value}<br />
+            <b>Message</b>: {message.value}<br />
+            """
+
+            # Send the email
+            try:
+                requests.post(
+                    "https://api.postmarkapp.com/email",
+                    headers={
+                        "Accept": "application/json",
+                        "Content-Type": "application/json",
+                        "X-Postmark-Server-Token": postmark_api_key,
+                    },
+                    data=json.dumps(msg),
+                )
+                print("Email sent successfully!")
```

Comment:
It is strange that our linting does not catch this, it used to not allow this.

Reply:
Huh, indeed.